examples/models/llama/source_transformation/sdpa.py

```diff
@@ -47,20 +37,21 @@ def forward(
         seqlen,
         mask,
     ):
+        q = q.transpose(1, 2)  # (bs, seqlen, n_local_heads, head_dim)
```

Comment:
I just thought about it again, and adding this transpose here and also before in the `llama_transformer.py` so that we can share code for kv_cache.py (this is the reason right?) doesn't really make sense since we are using a custom export-friendly KV cache already anyways: https://github.com/pytorch/executorch/blob/main/extension/llm/modules/kv_cache.py#L13